packages/vm/src/errors.rs

```diff
@@ -99,28 +102,120 @@ impl From<wasmer_runtime_core::cache::Error> for VmError {
     }
 }
 
+impl From<wasmer_runtime_core::error::CompileError> for VmError {
```

Comment:
Ahh.. this is how you got `?` to auto-convert eg. `CompileError` into `VmError`.
That is super clean!

Reply:
Jap, learned this one from Reuven